go/vt/vttablet/tabletserver/schema/load_table_test.go

```diff
@@ -96,6 +98,13 @@ func TestLoadTableSequence(t *testing.T) {
 	if !reflect.DeepEqual(table, want) {
 		t.Errorf("Table:\n%#v, want\n%#v", table, want)
 	}
+
+	table.SequenceInfo.NextVal = 10
+	table.SequenceInfo.LastVal = 5
+	table.SequenceInfo.Reset()
+	if !reflect.DeepEqual(table, want) {
```

Comment:
nit: can switch to using `utils.Match` here.

Reply:
Done